features/post-with-keys.js

```diff
@@ -0,0 +1,19 @@
+var page = function () {
+    if (document.URL.startsWith("https://scratch.mit.edu/discuss")) {
+        return "f"; // F = forums
+    } else {
+        return "e"; // E = else (not forums)
+    }
+}
+
+(function() {
```

Comment:
Why is the code itself a function? It's acting like a bookmarklet, when it's in a file.